modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheOperationContext.java

```diff
@@ -70,19 +73,13 @@
      */
     public CacheOperationContext() {
         skipStore = false;
-
         subjId = null;
-
         keepBinary = false;
-
         expiryPlc = null;
-
         noRetries = false;
-
         recovery = false;
-
+        readRepair = false;
```

Comment:
empty line required

Reply:
please recheck, seems that in strictly not required.